engine/installation/linux/debian.md

```diff
@@ -287,13 +312,14 @@ installing Docker.
     Use this command to install the latest version of Docker:
 
     ```bash
-    $ sudo apt-get install docker
+    $ sudo apt-get install docker-engine
```

Comment:
Actually I checked that when installing `docker-engine` one actually installs Docker CE from the edge channel. So not an older version. For how long this is going to be maintained is unknown to me but should be answered.

Personally, on my Raspberry Pi 2 which is running Raspbian Jessie, I have used the Debian installation instruction for armhf. I did not use the convenience script nor the old URL specific to Raspbian. Therefore I have installed Docker CE and I chose the stable channel. It is now working for roughly a month and it is very stable. I am not sure why we would need a Raspbian dedicated section in the Debian installation, I think we should use the same approach as for Debian on armhf, unless there is a specific package made by Docker for Raspbian. This needs to be confirmed by a Docker responsible.

Why didn't I remove the Raspbian section altogether? Well, I'm obviously not from Docker but a contributor. I do not know if Docker provides a specific Raspbian package or not. I also did not want to delete a complete section without this knowledge.